src/inventory/mod.rs

```diff
@@ -25,9 +25,13 @@ pub trait Inventory {
 
     fn name(&self) -> Option<&String>;
 
-    fn get_item(&self, slot: u16) -> Option<Item>;
+    fn get_action_number(&self) -> i16;
 
-    fn set_item(&mut self, slot: u16, item: Option<Item>);
+    fn set_action_number(&mut self, action_number: i16);
```

Comment:
This name seems a bit ambigous. It would be nice if we could somehow indicate that the action is the action our client is performing and not some inherent property of the inventory or something that is influenced by other entities/players

Reply:
I've added docs above this now to make it more clear. I can change the name if you would like, but I'm not sure which name is better. The name used in minecraft-data is "action number", but I can switch it to state id which is the name used in wiki.vg. I can also add client to the name, like `set_client_state_id` or `set_client_action_number`.

Reply:
I really like the `client` prefix but i don't particularly care about the suffix, although i'd say `_state_id` sounds a bit better and is a bit more descriptive, at least the `id` part instead of `number` is an improvement

Reply:
I've updated this to `set_client_state_id`